csFastFloat/Utils/Utils.cs

```diff
@@ -201,16 +196,17 @@ internal unsafe static bool strncasecmp(byte* input1, ReadOnlySpan<byte> input2,
       return (running_diff == 0) || (running_diff == 32);
     }
 
-    internal unsafe static bool strncasecmp(byte* input1, byte* input2, int length)
-    {
-      int running_diff = 0;
+    // Unreferenced
```

Comment:
I am puzzled because I see Utils.strncasecmp(first, infinity_string, 8) being called in this PR and yet it is getting commented out... ???